shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -330,6 +331,17 @@ public InputConnection createInputConnection(
     }
     outAttrs.imeOptions |= enterAction;
 
+    String[] imgTypeString = new String[] {
+      "image/png",
+      "image/bmp",
+      "image/jpg",
+      "image/tiff",
+      "image/gif",
+      "image/jpeg",
+      "image/webp"
+    };
+    EditorInfoCompat.setContentMimeTypes(outAttrs, imgTypeString);
```

Comment:
I noticed that on Android, if try I use Gboard to insert a gif into a field that doesn't support images, it gives me a message saying, "Google does not support image insertion here".  Is that message based on this mime type list?  If so, maybe we need a way for Flutter developers to specify that a field does not support images.

<img height="300" src="https://user-images.githubusercontent.com/389558/127544477-b2378282-f0f2-4686-a052-64b86584def0.jpg" />

Reply:
I also tried this on a Flutter app (before your changes) and I get the same message.  We might want to include a way to support this original behavior.  The first thing that comes to mind is adding a contentMimeTypes parameter to EditableText and passing it through to here, would that work?

Reply:
As far as I know that message is only based on whether the `contentCommit` hook is overriden in the `InputClient`. It could be based off the `mimeType` list, in which case I agree finding a way to pass those to there would be a good idea.

Reply:
@justinmc so apparently the mime types do nothing in @zlshames's testing... how should we approach this then? Should we just remove this code entirely? 

I don't really know why it doesn't affect the content insertion, it seems to me that we have everything set up correctly. If we indeed haven't made any mistakes, imho the best option would be to let the dev handle their desired `mimeType`s `onContentCommit` and choose how to notify their users that the content they are inserting is unsupported. Thinking on the iOS side as well, I am not 100% sure you can control the `mimeType` of items being pasted into an editable text field -so it may be better to do it this way for the sake of consistency.